Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackFormViewModel.swift

```diff
@@ -245,11 +242,27 @@ class SentryUserFeedbackFormViewModel: NSObject {
     lazy var logoViewWidthConstraint = sentryLogoView.widthAnchor.constraint(equalToConstant: logoWidth * config.scaleFactor)
     lazy var fullNameTextFieldHeightConstraint = fullNameTextField.heightAnchor.constraint(equalToConstant: formElementHeight * config.scaleFactor)
     lazy var emailTextFieldHeightConstraint = emailTextField.heightAnchor.constraint(equalToConstant: formElementHeight * config.scaleFactor)
-    lazy var addScreenshotButtonHeightConstraint = addScreenshotButton.heightAnchor.constraint(equalToConstant: formElementHeight * config.scaleFactor)
     lazy var removeScreenshotButtonHeightConstraint = removeScreenshotButton.heightAnchor.constraint(equalToConstant: formElementHeight * config.scaleFactor)
     lazy var submitButtonHeightConstraint = submitButton.heightAnchor.constraint(equalToConstant: formElementHeight * config.scaleFactor)
     lazy var cancelButtonHeightConstraint = cancelButton.heightAnchor.constraint(equalToConstant: formElementHeight * config.scaleFactor)
-    lazy var screenshotImageAspectRatioConstraint = screenshotImageView.widthAnchor.constraint(equalTo: screenshotImageView.heightAnchor)
+    lazy var screenshotImageAspectRatioConstraint = {
+        let aspectRatio: CGFloat
+        if let screenshot = self.screenshot {
+            if screenshot.size.height == 0 {
```

Comment:
`m`: Maybe it's safer to check for both height and width not being 0.

Reply:
This is just to avoid a div0 crash. If we have an image with 0 width, presentation is going to be borked regardless of whether or aspect ratio is 0 or 1